content/en/docs/tasks/observability/logs/otel-provider/index.md

```diff
@@ -0,0 +1,203 @@
+    port: 4317
+{{< /text >}}
+
+The final configuration should look something like:
+
+{{< text yaml >}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: istio
+  namespace: istio-system
+data:
+  mesh: |-
+    accessLogFile: /dev/stdout
+    defaultConfig:
+      discoveryAddress: istiod.istio-system.svc:15012
+      proxyMetadata: {}
+      tracing:
+        zipkin:
+          address: zipkin.istio-system:9411
+    enablePrometheusMerge: true
+    extensionProviders:
+    - name: otel
+      envoyOtelAls:
+        service: otel-collector.istio-system.svc.cluster.local
+        port: 4317
+    rootNamespace: istio-system
+    trustDomain: cluster.local
+  meshNetworks: 'networks: {}'
+{{< /text >}}
+
+Next, add a Telemetry resource that tells Istio to send access logs to the OpenTelemetry collector.
+
+{{< text bash >}}
+$ cat <<EOF | kubectl apply -n default -f -
+apiVersion: telemetry.istio.io/v1alpha1
+kind: Telemetry
+metadata:
+  name: sleep-logging
+spec:
+  selector:
+    matchLabels:
+      app: sleep
+  accessLogging:
+    - providers:
+      - name: otel
+EOF
+{{< /text >}}
+
+The above example uses the `otel` access log provider, and we do not configure anything other than default settings.
+
+Similar configuration can also be applied on an individual namespace, or to an individual workload, to control logging at a fine grained level.
+
+For more information about using the Telemetry API, see the [Telemetry API overview](/docs/tasks/observability/telemetry/).
+
+### Using Mesh Config
+
+If you used an `IstioOperator` CR to install Istio, add the following field to your configuration:
+
+{{< text yaml >}}
+spec:
+  meshConfig:
+    accessLogFile: /dev/stdout
+    extensionProviders:
+    - name: otel
+      envoyOtelAls:
+        service: otel-collector.istio-system.svc.cluster.local
+        port: 4317
+    defaultProviders:
+      accessLogging:
+      - envoy
+      - otel
+{{< /text >}}
+
+Otherwise, add the equivalent setting to your original `istioctl install` command, for example:
+
+{{< text syntax=bash snip_id=none >}}
+$ istioctl install -f <your-istio-operator-config-file>
+{{< /text >}}
+
+## Default access log format
+
+Istio will use the following default access log format if `accessLogFormat` is not specified:
+
+{{< text plain >}}
+[%START_TIME%] \"%REQ(:METHOD)% %REQ(X-ENVOY-ORIGINAL-PATH?:PATH)% %PROTOCOL%\" %RESPONSE_CODE% %RESPONSE_FLAGS% %RESPONSE_CODE_DETAILS% %CONNECTION_TERMINATION_DETAILS%
+\"%UPSTREAM_TRANSPORT_FAILURE_REASON%\" %BYTES_RECEIVED% %BYTES_SENT% %DURATION% %RESP(X-ENVOY-UPSTREAM-SERVICE-TIME)% \"%REQ(X-FORWARDED-FOR)%\" \"%REQ(USER-AGENT)%\" \"%REQ(X-REQUEST-ID)%\"
+\"%REQ(:AUTHORITY)%\" \"%UPSTREAM_HOST%\" %UPSTREAM_CLUSTER% %UPSTREAM_LOCAL_ADDRESS% %DOWNSTREAM_LOCAL_ADDRESS% %DOWNSTREAM_REMOTE_ADDRESS% %REQUESTED_SERVER_NAME% %ROUTE_NAME%\n
+{{< /text >}}
+
+The following table shows an example using the default access log format for a request sent from `sleep` to `httpbin`:
```

Comment:
On my computer, this table is very wide and overwrites the right hand column.

Reply:
Other than the table, this LGTM.

Reply:
Other than the table, this LGTM.

Reply:
this table come from another page, and I do not change anything, I have no idea about what's happen on your compute.
can we merged this first, let's see what happen to https://preliminary.istio.io/?

Reply:
https://deploy-preview-10977--preliminary-istio.netlify.app/latest/docs/tasks/observability/logs/otel-provider/

I'm OK with merging it and fixing the rendering bug separately.  (I don't think you've done anything wrong.)

Reply:
the page looks good.

Reply:
Here's an example on what I see. As Craig mentioned, we can fix this rendering later.
<img width="1635" alt="Screen Shot 2022-03-15 at 7 57 12 AM" src="https://user-images.githubusercontent.com/10537847/158382651-1fd1aaf6-9d7a-4096-bf94-bd460caa1d78.png">

Reply:
this's a common issue, see https://istio.io/latest/docs/tasks/observability/logs/access-log/
I can take a look at this later this week.